cliquet/cache/memory.py

```diff
@@ -46,9 +51,13 @@ def get(self, key):
         return self._store.get(key)
 
     def delete(self, key):
+        key = self.prefix + key
         self._ttl.pop(key, None)
         self._store.pop(key, None)
 
 
 def load_from_config(config):
-    return Cache()
+    settings = config.get_settings()
+    prefix = settings['cache_prefix']
+    backend = Cache(prefix=prefix)
+    return backend
```

Comment:
You can probably do ``return Cache(prefix=settings['cache_prefix'])`` and save some intermediary variables.